design/use_cases/indoor_localization_use_case.md

```diff
@@ -0,0 +1,15 @@
+# Indoor Localization
+As a Robot user I want my robot to know its location on a given map of an indoor area so that it can move around the area
+
+## More details
+- Why is this needed?
+   - This is needed for indoor robot navigation in most (all?) cases
+   - Example: a courier robot in a logistics warehouse
+
+- What is the expected user interaction?
+   - The user should be able to specify a map to use and a location on that map for the robot
```

Comment:
In most cases it will be but we shouldn't limit the design to only handle the initial position. Sometimes the robot may need to be given an adjusted position, for example if it is in a very large space it may not have a nearby point of reference.

Reply:
Sounds like then there is an option for specifying a relative location. What kind of semantics are associated with these potential anchor locations? For example, "meet me by the water cooler." It sounds like, during the mapping process, that the Robot User would need to identify names/IDs for these significant locations.